src/plugins/autonomy/ScrimmageOpenAIAutonomy/OpenAIObservations.cpp

```diff
@@ -108,6 +108,17 @@ pybind11::object OpenAIObservations::update_observation(size_t num_entities) {
             observation_space.attr("spaces").cast<py::list>();
         py::list observation_list = observation.cast<py::list>();
 
+        // Get python len() function
+        py::object len_ = py::module::import("builtins").attr("len");
+
+        // Number of entities has shrunk
+        while (len_(observation_list).cast<size_t>() > ext_sensor_vec_.size()) {
```

Comment:
pybind11 has a built-in len function that we can use. See [here](https://github.com/pybind/pybind11/blob/master/include/pybind11/pytypes.h#L1331)